pkg/kclient/binding.go

```diff
@@ -0,0 +1,142 @@
+package kclient
+
+import (
+	"context"
+	"errors"
+
+	sboApi "github.com/redhat-developer/service-binding-operator/apis/binding/v1alpha1"
+	kerrors "k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/api/meta"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+)
+
+const (
+	ServiceBindingGroup    = "binding.operators.coreos.com"
+	ServiceBindingVersion  = "v1alpha1"
+	ServiceBindingKind     = "ServiceBinding"
+	ServiceBindingResource = "servicebindings"
+	BindableKindsResource  = "bindablekinds"
+)
+
+// IsServiceBindingSupported checks if resource of type service binding request present on the cluster
+func (c *Client) IsServiceBindingSupported() (bool, error) {
+	// Detection of SBO has been removed from issue https://github.com/redhat-developer/odo/issues/5084
+	return c.IsResourceSupported(ServiceBindingGroup, ServiceBindingVersion, ServiceBindingResource)
+}
+
+// GetBindableKinds returns BindableKinds of name "bindable-kinds".
+// "bindable-kinds" is the default resource provided by SBO
+func (c *Client) GetBindableKinds() (sboApi.BindableKinds, error) {
+	if c.DynamicClient == nil {
+		return sboApi.BindableKinds{}, nil
+	}
+
+	var (
+		unstructuredBK *unstructured.Unstructured
+		bindableKind   sboApi.BindableKinds
+		err            error
+	)
+
+	gvr := schema.GroupVersionResource{Group: ServiceBindingGroup, Version: ServiceBindingVersion, Resource: BindableKindsResource}
```

Comment:
You could define gvr as a package constant, and also gvk.

Reply:
> `Const initializer 'schema.GroupVersionResource{ Group: "", Version: "", Kind: "", }' is not a constant`

Cannot do that. I could define a global variable, but I am not sure very sure about it, since it is vulnerable to change.

Reply:
Or you can use the value GroupVersionResource defined in https://github.com/redhat-developer/service-binding-operator/blob/master/apis/binding/v1alpha1/groupversion_info.go